docker-compose.yml

```diff
@@ -34,7 +34,12 @@ services:
   account-store:
     build:
       context: ./apps/funding-service-design-account-store
-    command: [ "sh", "-c", "flask db upgrade && python -m debugpy --listen 0.0.0.0:5678 -m wsgi" ]
+    command:
+      bash -c "
+      flask db upgrade && \
+      python -m invoke seed_local_account_store && \
```

Comment:
This does work, but I'm mildly confused because I am sure in some places Marc and I have found that invoke tasks expected to be hyphenated rather than underscored 🤔  maybe we're inconsistent about that config in places?

Reply:
I think we are just inconsistent

Reply:
Fixed - re-review please!